packages/react-dom/src/__tests__/ReactDOM-test.js

```diff
@@ -50,8 +56,11 @@ describe('ReactDOM', () => {
     }
 
     document.body.appendChild(container);
+    const root = ReactDOMClient.createRoot(container);
     try {
-      ReactDOM.render(<Parent />, container);
+      await act(() => {
+        root.render(<Parent />);
+      });
```

Comment:
Wonder if we should have a test where the click/focus/etc inside of act, to show what happens if you click before the render completes, since it's async in createRoot? Maybe we have a test for that somewhere.

Reply:
Yeah, I don't have a good understanding of what tests exists. I'll keep this PR focused to just removing `.render()` calls.